llvm/test/CodeGen/AMDGPU/print-pipeline-passes.AFLCustomIRMutator.ll

```diff
@@ -0,0 +1,11 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx90a -O1 < %s | FileCheck -check-prefix=GCN %s
```

Comment:
Test file name doesn't make sense, can you add this case to one of the existing div-rem IR expansion tests

Reply:
but if you run the pass in isolation it should work?

Reply:
You can just move it to amdgpu-codegenprepare-idiv.ll